src/value-parser.js

```diff
@@ -387,6 +395,30 @@ function readNChars(parser, dataLength, callback) {
   }
 }
 
+function readText(parser, dataLength, codepage, callback) {
```

Comment:
@mgetz since `readNText`, `readText` and  `readImage` are exactly same as `readChars`, `readNChars` and `readBinary`, except for the null check, it would be good to get rid of redundant function. How about passing additional parameter to the original methods ?
```node
function readBinary(parser, dataLength, nullType /*receives PLP_NULL or NULL*/, callback) {
  if (dataLength === nullType) {..}
  ...
}
```
Sorry for asking you to update the PR again 😅

Reply:
No problem at all. Had considered it when I added those two extra functions. :-)